test/cases/compile_errors/comptime_var_referenced_by_type.zig

```diff
@@ -21,5 +21,6 @@ comptime {
 // error
 //
 // :7:16: error: captured value contains reference to comptime var
-// :16:30: note: 'wrapper.ptr' points to comptime var declared here
+// :7:16: note: 'wrapper' points to '@as(*const tmp.Wrapper, @ptrCast(&v0)).*', where
+// :16:5: note: 'v0.ptr' points to comptime var declared here
```

Comment:
Ooh, nice side effect!

Reply:
oh really? I thought it was a slightly less good but still acceptable error message

Reply:
I agree with that, I misread this when I first looked at the diff, hence my edit; my bad!